test/e2e/scheduling/priorities.go

```diff
@@ -113,12 +113,24 @@ var _ = framework.KubeDescribe("SchedulerPriorities [Serial]", func() {
 		createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.6)
 
 		By("Create an RC with the same number of replicas as the schedualble nodes. One pod should be scheduled on each node.")
-		rc := createRC(ns, "scheduler-priority-selector-spreading", int32(len(nodeList.Items)), map[string]string{"name": "scheduler-priority-selector-spreading"}, f, podRequestedResource)
```

Comment:
@k82cn There is another place use createRC and  need create a 0 replica RC first, I want to keep it here.